keyboards/misterknife/knife66_iso/info.json

```
@@ -0,0 +1,84 @@
+{
+    "keyboard_name": "Knife66_iso",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 16.5,
+    "height": 5.25,
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                { "x": 0, "y": 0 },
+                { "x": 1, "y": 0 },
+                { "x": 2, "y": 0 },
+                { "x": 3, "y": 0 },
+                { "x": 4, "y": 0 },
+                { "x": 5, "y": 0 },
+                { "x": 6, "y": 0 },
+                { "x": 7, "y": 0 },
+                { "x": 8, "y": 0 },
+                { "x": 9, "y": 0 },
+                { "x": 10, "y": 0 },
+                { "x": 11, "y": 0 },
+                { "x": 12, "y": 0 },
+                { "x": 13, "y": 0 },
+                { "x": 14, "y": 0 },
+                { "x": 15.5, "y": 0 },
+                { "x": 0, "y": 1, "w": 1.5 },
+                { "x": 1.5, "y": 1 },
+                { "x": 2.5, "y": 1 },
+                { "x": 3.5, "y": 1 },
+                { "x": 4.5, "y": 1 },
+                { "x": 5.5, "y": 1 },
+                { "x": 6.5, "y": 1 },
+                { "x": 7.5, "y": 1 },
+                { "x": 8.5, "y": 1 },
+                { "x": 9.5, "y": 1 },
+                { "x": 10.5, "y": 1 },
+                { "x": 11.5, "y": 1 },
+                { "x": 12.5, "y": 1 },
+                { "x": 13.5, "y": 1, "w": 1.5 },
+                { "x": 15.5, "y": 1 },
+                { "x": 0, "y": 2, "w": 1.75 },
+                { "x": 1.75, "y": 2 },
+                { "x": 2.75, "y": 2 },
+                { "x": 3.75, "y": 2 },
+                { "x": 4.75, "y": 2 },
+                { "x": 5.75, "y": 2 },
+                { "x": 6.75, "y": 2 },
+                { "x": 7.75, "y": 2 },
+                { "x": 8.75, "y": 2 },
+                { "x": 9.75, "y": 2 },
+                { "x": 10.75, "y": 2 },
+                { "x": 11.75, "y": 2 },
+                { "x": 12.75, "y": 2, "w": 2.25 },
+                { "x": 15.5, "y": 2 },
+                { "x": 0, "y": 3, "w": 1.25 },
+                { "x": 1.25, "y": 3 },
+                { "x": 2.25, "y": 3 },
+                { "x": 3.25, "y": 3 },
+                { "x": 4.25, "y": 3 },
+                { "x": 5.25, "y": 3 },
+                { "x": 6.25, "y": 3 },
+                { "x": 7.25, "y": 3 },
+                { "x": 8.25, "y": 3 },
+                { "x": 9.25, "y": 3 },
+                { "x": 10.25, "y": 3 },
+                { "x": 11.25, "y": 3 },
+                { "x": 12.25, "y": 3, "w": 1.75 },
+                { "x": 15.5, "y": 3 },
+                { "x": 14.25, "y": 3.25 },
```

Need to switch these two keys around for the QMK Configurator support to work correctly.

```suggestion
                { "x": 14.25, "y": 3.25 },
                { "x": 15.5, "y": 3 },
```